src/Controller/CommentController.php

```diff
@@ -0,0 +1,86 @@
+use Symfony\Component\Security\Core\Security;
+use Symfony\Component\Validator\Validator\ValidatorInterface;
+use Symfony\Component\Validator\Constraints\NotBlank;
+use Symfony\Component\HttpFoundation\Response;
+use App\Entity\Comment;
+use App\Entity\Article;
+use App\Validator\Constraints\IsCurseFree;
+use Doctrine\ORM\ORMException;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Security as AccessGuard;
+
+class CommentController extends AbstractController
+{
+    /**
+     * @Route("/comment/add", name="new_comment")
+     */
+    public function addComment(Request $request, Security $security, ValidatorInterface $validator): Response
+    {
+        $comment = new Comment();
+
+        $comment
+        ->setContent($request->query->get('commentContent'))
+        ->setArticle($this->getDoctrine()->getRepository(Article::class)
+        ->find($request->query->get('articleId')));
+        $user = $security->getUser();
+        if (null !== $user) {
+            $comment
+            ->setName($user->getName().' '.$user->getSurname())
+            ->setEmail($user->getEmail());
+        } else {
+            $comment
+            ->setName($request->query->get('commentAuthor'))
+            ->setEmail($request->query->get('commentAuthorEmail'));
+        }
+
+        $violations = $validator->validate($comment);
+        $article = $this->getDoctrine()->getManager()->find(Article::class, $request->query->get('articleId'));
+
+        if ($violations->has(0)) {
+            $errors = [];
+            for ($i = 0; $i < $violations->count(); $i++) {
+                $errors[$violations->get($i)->getPropertyPath()] = $violations->get($i)->getMessage();
+            }
+
+            return $this->render('articles/show.html.twig', [
+                'article' => $article,
+                'errors' => $errors,
+            ]);
+        }
+
+        try {
+            $manager = $this->getDoctrine()->getManager();
+            $manager->persist($comment);
+            $manager->flush();
+        } catch (ORMException $exception) {
+            $this->addFlash('dbFailure', 'Nie udało się dodać komentarza. Spróbuj ponownie');
+            return $this->render('articles/show.html.twig', ['article' => $article]);
+        }
+
+        $this->addFlash('success', 'Dodano komentarz');
+
+        return $this->redirectToRoute('article_show', ['id' => $request->query->get('articleId')]);
+    }
+
+    /**
+     * @Route("/comment/delete", name="delete_comment")
+     * @AccessGuard("is_granted('ROLE_ADMIN')")
+     */
+    public function deleteComment(Request $request): Response
+    {
+        $entityManager = $this->getDoctrine()->getManager();
+        $comment = $entityManager->find(Comment::class, $request->get('commentId'));
+        $entityManager->remove($comment);
+        $entityManager->flush();
+
+        return $this->redirectToRoute('article_show', ['id' => $request->query->get('articleId')]);
```

Comment:
dostaję 500   
`Parameter "id" for route "article_show" must match "\d+" ("" given) to generate a corresponding URL.`

Nawet jak naprawisz to nie ma sensu przekazywać `id` artykułu, ponieważ możesz pobrać artykuł i jego id poprzez `$comment->getArticle()->getId()`. Chociaż z drugiej strony dodanie tej informacji do query stringa spowoduje, że nie będziemy musieli wykonać zapytania do bazy danych o artykuł (ale wątpię, aby to był Twój cel kiedy to pisałeś). Jeśli nie wiesz o czym mówię to zapoznaj się z pojęciem **lazy loading** i jak wygląda pole $article obiektu $comment przed użyciem metody `getArticle()`, a jak po użyciu tej metody).